src/hotspot/share/c1/c1_InstructionPrinter.cpp

```diff
@@ -239,7 +239,7 @@ void InstructionPrinter::print_stack(ValueStack* stack) {
     output()->cr();
     fill_to(start_position, ' ');
     output()->print("locks [");
-    for (int i = i = 0; i < stack->locks_size(); i++) {
+    for (int i = 0; i < stack->locks_size(); ++i) {
```

Comment:
I would prefer to keep `i++`, as in old code, unless it fixes something I don't quite see.

Reply:
Done. Can you, please, create an issue, so I can mention it here?

Reply:
I did already, see https://github.com/openjdk/jdk/pull/1519#issuecomment-738180725.

Reply:
Oh, sorry! I guess, haven't completely got back from the vacation :-)

Reply:
Pipeline is on the way.